server/controllers/match.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose')
+
+// add a match and return match ID
+function addMatch() {
+
+  var Match = mongoose.model('matches');
+  var match = new Match();
+  var matchID = null;
+  match.save(function(err, matchInfo) {
+    if (err) throw err;
+    console.log('Match saved, ID: ' + matchInfo.ID);
+    matchID = matchInfo.ID;
```

Comment:
```suggestion
    matchID = matchInfo._id;
```
In Mongo ids are accessed at the `_id` property

Reply:
Changed it so that we are now accessing the match id using '_id'